assets/src/components/map/controls/StreetViewSwitch.tsx

```diff
@@ -110,23 +111,21 @@ export const StreetViewSwitch = ({
       >
         Street View
       </label>
-      <div className="form-check form-switch c-street-view-switch__input">
-        <input
-          id={id}
-          className="form-check-input"
-          type="checkbox"
-          role="switch"
-          checked={streetViewEnabled}
-          onChange={() => {
-            // since the value is being toggled, the new value will be the opposite of the current value
-            fullStoryEvent("Dedicated street view toggled", {
-              streetViewEnabled_bool: !streetViewEnabled,
-            })
+      <Form.Check
+        type="switch"
+        id={id}
+        className="c-street-view-switch__input"
+        role="switch"
```

Comment:
question: I would've expected that `type="switch"` would set this, do you know of `role` is required to set it to `role=switch`?